frontend/src/composables/useShootSpec.js

```diff
@@ -119,11 +121,20 @@ export function useShootSpec (shootItem, options = {}) {
   })
 
   const isCustomShootDomain = computed(() => {
-    return some(shootDnsProviders.value, 'primary')
+    return !!shootDnsPrimaryProvider.value
   })
 
-  const shootDnsProviders = computed(() => {
-    return get(shootSpec.value, 'dns.providers')
+  const shootDnsPrimaryProvider = computed(() => {
+    return find(shootSpec.value.dns?.providers, { primary: true })
+  })
+  const shootExtensionDnsProviders = computed(() => {
+    const extensionDns = find(shootSpec.value.extensions, { type: 'shoot-dns-service' })
```

Comment:
we typically use the list-style for the lodash `find` function, instead of the object
however I must admit that the object style feels more natural (why do we use the list style again? :D)

Reply:
changed to property shorthand